docs/api/1.x/accounts.rst

```diff
@@ -34,6 +34,21 @@ Add the following settings to the ``.ini`` file:
     kinto.account_create_principals = system.Everyone
 
 
+You can use the ``create-user`` command to create an admin:
+
+.. code-block:: bash
+
+    kinto create-user --ini /etc/kinto.ini --username admin --password ThisIsN0tASecurePassword
```

Comment:
I think the most common way is to rely on prompt for password no?

Reply:
That's the case if you don't provide any arguments, however OPS will use SOPS and ENV variables so they want to be able to do: `into create-user --ini /etc/kinto.ini --username $USERNAME --password $PASSWORD`